pom.xml

```diff
@@ -214,9 +214,6 @@
             <resource>
                 <directory>src/main/resources</directory>
             </resource>
-            <resource>
-                <directory>target/metamodel</directory>
```

Comment:
My ide was constantly putting this folder as test resource instead of source. Since th Qclasses are source, or it doesnt compile